tests/test_cases/statfs.py

```diff
@@ -0,0 +1,28 @@
+#!/bin/python2
+# Check if statfs.c runs correctly in native as it does within Lind
+import sys 
+import re
+
+# Receive result
+lind_result = sys.argv[1]
+host_result = sys.argv[2]
+
+# Extract output
+# sample output: Filesystem type: beefcOde
+match = re.search(r'Filesystem type: (\w+)', lind_result)
+if match:
+    lind_string = match.group (1)
+# Expect lind output: beefcOde
+if lind_string != "beefc0de":
+    print "Lind info: ", lind_string
+    exit (-1)
+
+# Extract output
+# sample output: Filesystem type: 794c7630
+match2 = re. search (r'Filesystem type: (\w+)', host_result)
```

Comment:
```suggestion
match2 = re.search(r'Filesystem type: (\w+)', host_result)
```

Could you remove spaces like this throughout the file?